packages/storybook/package.json

```diff
@@ -21,6 +21,7 @@
   "devDependencies": {
     "@storybook/addon-a11y": "8.0.4",
     "@storybook/addon-essentials": "8.0.4",
+    "@storybook/addon-links": "^8.0.8",
```

Comment:
never use "^" and set the same version as other
`"@storybook/addon-links": "8.0.4",`